checklist.md

```diff
@@ -36,9 +36,10 @@ from app.main import (
 Bad example:
 
 ```python
-from app.main import KnightWithArmour, \
-    KnightWithoutArmour, \
-    SuperHeroKnightWithGun
+from app.main import KnightWithArmour,
+
+KnightWithoutArmour,
+SuperHeroKnightWithGun
```

Comment:
Look like useless changes

Reply:
Roll back these changes to the initial state.